sale_payment_method_automatic_workflow/model/automatic_workflow_job.py

```diff
@@ -37,6 +41,21 @@ def _reconcile_invoices(self):
 
     @api.model
     def run(self):
+        self._autopay()
         res = super(AutomaticWorkflowJob, self).run()
         self._reconcile_invoices()
         return res
+
+    @api.model
+    def _get_domain_for_sale_autopayment(self):
+        return [('state', '=', 'draft'),
```

Comment:
It is intentional that this will not get `open` orders? Do you think this could interfere with the workflow that confirms orders in `sale_automatic_workflow`?